UM/Application.py

```diff
@@ -101,6 +101,12 @@ def __init__(self, name: str, version: str, build_type: str = "", **kwargs):
 
         self._operation_stack = OperationStack(self.getController())
 
+        # Ensure configuration folder has a folder for themes
+        try:
+            os.makedirs(os.path.join(Resources.getStoragePath(Resources.Resources), "themes"))
```

Comment:
I'll test without it and will see if the folder gets created. I added it preemptively because I remembered people saying they could not find the "plugins" folder in the configuration folder.